packages/flutter/lib/src/widgets/autofill.dart

```diff
@@ -137,6 +174,18 @@ class AutofillGroup extends StatefulWidget {
   /// {@macro flutter.widgets.child}
   final Widget child;
 
+  /// The [AutofillContextAction] to be run when this [AutofillGroup] is the
+  /// topmost [AutofillGroup] and it's being disposed, in order to clean up the
+  /// current autofill context.
+  ///
+  /// {@macro flutter.services.autofill.autofillContext}
+  ///
+  /// Defaults to [AutofillContextAction.commit], which prompts the platform to
+  /// save the user input and destroy the current autofill context. No
+  /// autofill context related actions will be automatically taken when
```

Comment:
autofill context related actions will be automatically taken  => no action will be taken